__tests__/pathfinding.service.test.js

```diff
@@ -70,6 +71,106 @@ describe("Find the shortest path on a given floor", () => {
       POIInfo.filter(({ displayName }) => displayName === "H961-9")[0].location
     );
     expect(shortest.length).toBe(1);
-    expect(travelPathToLinePath(shortest)).toEqual(testData.h96119toH9619);
+    expect(shortest).toEqual(testData.h96119toH9619);
+  });
+
+  it("should return a path that uses stairs as connectors", () => {
+    constants.buildingFloors = buildingFloors.push({
+      id: 3,
+      buildingId: BuildingId.H,
+      level: 3,
+      bounds: null,
+      image: null,
+      travelNodes: [
+        {
+          id: 0,
+          location: {
+            latitude: 45.497411422038525,
+            longitude: -73.57930387743083
+          },
+          children: [1]
+        },
+        {
+          id: 1,
+          location: {
+            latitude: 45.49735509062119,
+            longitude: -73.57918419446206
+          },
+          children: [0]
+        }
+      ]
+    });
+
+    constants.POIInfo.push({
+      id: "8dd47eee-23b4-437c-bd74-aad3876f83dc",
+      displayName: "Stairs 1",
+      description: "",
+      location: {
+        latitude: 45.497354,
+        longitude: -73.578713
+      },
+      buildingId: BuildingId.H,
+      level: 3,
+      category: POICategory.Stairs
+    });
```

Comment:
I'm a little concerned that mock data is being pushed into constants. Can this be in `__testData__` or something similar?

Reply:
We have to modify the actual objects in `src/constants` since the algorithms use it and we didn't want to duplicate such a large object. We could probably move the pushes into a `before` and `after` block and the new data to `__testData__` How does that sound?

Reply:
Maybe clone the data in your test file?

Reply:
We came up with something. Check out the new code since this file is outdated. Let me know what you think

Reply:
Nice, looks resolved to me!